agent/src/com/jonnyzzz/teamcity/plugins/node/agent/NodeDetector.kt

```diff
@@ -74,15 +85,26 @@ class NodeToolsDetector(events: EventDispatcher<AgentLifeCycleListener>,
       }
       else -> {
         LOG.info("$executable was not found or failed, exitcode: ${run.exitCode}")
-        LOG.info("StdOut: ${run.stdOut}")
-        LOG.info("StdErr: ${run.stdErr}")
+        if (!run.stdOut.isEmptyOrSpaces()) LOG.info("StdOut: ${run.stdOut}")
+        if (!run.stdErr.isEmptyOrSpaces()) LOG.info("StdErr: ${run.stdErr}")
       }
     }
   }
 
   init {
     events.addListener(object : AgentLifeCycleAdapter() {
       override fun beforeAgentConfigurationLoaded(agent: BuildAgent) {
+        setVerbose(true)
```

Comment:
It's better to have `doDetectTools` have a parameter to specify the way logging is performed. 
I do not like an idea to reconfigure logger in any way somewhere here